modules/nickserv/sendpass.c

```diff
@@ -198,11 +198,19 @@ ns_cmd_sendpass(struct sourceinfo *si, int parc, char *parv[])
 			else
 				command_success_nodata(si, _("Overriding MARK on the account %s."), entity(mu)->name);
 		}
-		logcommand(si, CMDLOG_ADMIN, "SENDPASS: \2%s\2", name);
 
 		newpass = random_string(16);
```

Comment:
This was already the case before this PR, but fwiw this probably should use `config_options.default_pass_length` instead of hardcoding 16